nano/test_common/system.cpp

```diff
@@ -563,7 +576,7 @@ void nano::test::system::stop ()
 
 nano::node_config nano::test::system::default_config ()
 {
-	nano::node_config config{ get_available_port (), logging };
+	nano::node_config config{ std::nullopt, logging };
```

Comment:
Historically, `get_available_port ()` was used to guarantee deterministic port allocation when running tests. Later it was modified to use ephemeral ports by default, but still make it possible to have them deterministic if needed. My understanding is that this won't allow that anymore

Reply:
I reinstated the previous behaviour.